customer-managed/azure/privatelink/README.md

```diff
@@ -0,0 +1,244 @@
+# Redpanda Azure private link
+
+
+### Getting Started
+
+#### Required tools:
+- rpk
+- terraform
+- Azure CLI
+- Curl
+- jq
+
+Once all tools are installed make sure you have logged into to the azure CLI
+
+```
+az login
+```
+
+## Provisioning Redpanda with Private Link
+
+To provision a redpanda with private link there is currently only API support with UX support coming soon. 
```

Comment:
suggestion, nonblocking: To provision a private Redpanda cluster with Private Link you must use the API. UI support is coming soon.